riverdriver/riverdatabasesql/river_database_sql_driver.go

```diff
@@ -364,6 +358,36 @@ func (e *Executor) JobList(ctx context.Context, query string, namedArgs map[stri
 	return mapSliceError(items, jobRowFromInternal)
 }
 
+// `database/sql` has an `sql.Named` system that should theoretically work for
+// named parameters, but neither Pgx or lib/pq implement it, so just use dumb
+// string replacement given we're only injecting a very basic value anyway.
+func replaceNamed(query string, namedArgs map[string]any) (string, error) {
+	for name, value := range namedArgs {
+		var escapedValue string
+
+		switch typedValue := value.(type) {
```

Comment:
I think a case for `[]string` is still missing.

Reply:
Yep, absolutely right. Just pushed one more update that should take care of slice handling.